draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -440,7 +446,7 @@
    </t>	
    <t>
 	The CI/T "invalidate" and "purge" trigger actions MUST be applied to all data acquired before dCDN begins the trigger 
-	processing (i.e. enters "active" state).
```

Comment:
Line 319 has a double-colon ("::") that needs to be change to a single colon.  (Sorry that this comment doesn't line up with that; there were no changes to this line in this commit.)